recipes/django-pyodbc-azure/meta.yaml

```diff
@@ -0,0 +1,48 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "django-pyodbc-azure" %}
+{% set version = "1.11.0.0" %}
+{% set sha256 = "1f41d46e914a3cfd301db9551970d50c0ecbf84b91085b6bb77706dfcff7ed2e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/michiya/django-pyodbc-azure/archive/1.11.0.0.zip
```

Comment:
Let's use a jinja template variable here instead of hardcoding the version number.